CHANGELOG.md

```diff
@@ -4,6 +4,98 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.15.0 - 2019-03-25
+
+### Added
+
+- We now initialize TLS certificates for etcd metrics over TCP on port
+  9979.  We also store etcd certificates in various `kube-system`
+  Secrets and ConfigMaps.
+- The Kubernetes client is now extracted from the release image's
+  `kube-client-agent` reference, replacing
+  `quay.io/coreos/kube-client-agent`.
+- CRI-O's pause image is now extracted from the release image's `pod`
```

Comment:
This doesn't make it clear that MCO uses these for control plane and compute nodes. Bootstrap node doesn't use it...

Reply:
> This doesn't make it clear that MCO uses these for control plane and compute nodes.

Reworded with 65607bb98 -> 620c0c0f6.  How does that look?